google/cloud/storage/blob.py

```diff
@@ -1881,6 +1881,22 @@ def update_storage_class(self, new_class, client=None):
     :rtype: str or ``NoneType``
 
     .. _RFC 4960: https://tools.ietf.org/html/rfc4960#appendix-B
+
+    Example:
+            Retrieve the crc32c hash of blob.
+
+            >>> from google.cloud import storage
+            >>> client = storage.Client()
+            >>> bucket = client.get_bucket("my-bucket-name")
+            >>> blob = bucket.blob('my-blob')
+
+            >>> blob.crc32c  #return None
```

Comment:
There should be a space after `#` (standard and so on)

Reply:
Any suggestion on description which i have mentioned above in comment

Reply:
In fact, I don't think, it's needed. Code speaks for itself. So, if I would have been doing this, I haven't comment it so detailed.